house/forms.py

```diff
@@ -0,0 +1,9 @@
+from django import forms
+from .models import House
```

Comment:
The reason the flake8 check failed was because this import was not used

Reply:
thanks